winter_messaging_transactional/producer/outbox/outbox_message.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Optional
+from uuid import UUID
+
+import sqlalchemy as sa
+from sqlalchemy import func
+from sqlalchemy.dialects import postgresql
+
+from winter_messaging_transactional.table_metadata import messaging_metadata
+
+
+@dataclass
+class OutboxMessage:
+    message_id: UUID
+    topic: str
+    type: str
+    body: str
+
+
+outbox_message_table = sa.Table(
+    'winter_outbox_message',
+    messaging_metadata,
+    sa.Column('id', sa.INTEGER, nullable=False, primary_key=True, autoincrement=True),
```

Comment:
We do we need integer id as primary key? `message_id: UUID` field already exists

@mikhaillazko 

I've found only one place where it is used https://github.com/WinterFramework/winter/pull/247/files#diff-632328ac9eebab0d7e5f4404dd017d71c42617e59e2e79e52d1e5aebd58fc280R31 

```
def select_unsent(self) -> Iterable[OutboxMessage]:
        query = select([
            outbox_message_table.c.message_id,
            outbox_message_table.c.topic,
            outbox_message_table.c.type,
            outbox_message_table.c.body,
        ]).where(
            outbox_message_table.c.sent_at.is_(None)
        ).order_by(outbox_message_table.c.id)
```

Reply:
we can't use `created_at` instead of 'id' because we have to guaranty stable order for messages, it's alternative for https://en.wikipedia.org/wiki/Lamport_timestamp